ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/icon/FontAwesome.java

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.icon;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.dom.client.Document;
+import com.google.gwt.dom.client.LinkElement;
+
+/**
+ * Font awesome style injector and set of predefined icons.
+ *
+ * @author Vitaliy Guliy
+ */
+public class FontAwesome {
```

Comment:
I think that this class is more related to the **che-core-ide-ui** module but not to the **che-core-ide-app**.
WDYT?